personal-practice/Dice.java

```diff
@@ -0,0 +1,23 @@
+import java.awt.*;
+import java.util.Scanner;
+import java.util.concurrent.Future;
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.Random;
+
+public class Dice {
```

Comment:
```suggestion
public final class Dice {
```

It's a good practice to make classes final.